NLP/BERT_PyTorch/classify_cola.py

```diff
@@ -0,0 +1,194 @@
+import torch
+import torch.nn as nn
+from random import randint, shuffle
+from random import random as rand
+from pytorch_pretrained_bert.tokenization import BertTokenizer
+import random
+import math
+import os
+import argparse
+import model_pretrain
+import wget
+import pandas as pd
+from utils import load
+
+device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+
+parser = argparse.ArgumentParser()
+# model config
+parser.add_argument('--dim', type=int, default=768)
+parser.add_argument('--max_len', type=int, default=512)
+parser.add_argument('--heads', type=int, default=12)
+parser.add_argument('--n_segs', type=int, default=2)
+
+parser.add_argument('--pretrain_file', type=str, required=True)
+
+parser.add_argument('--epochs', type=int, default=4)
+parser.add_argument('--batch_size', type=int, default=32)
+parser.add_argument('--lr', type=float, default=0.00002)
+parser.add_argument('--beta1', type=float, default=0.9)
+parser.add_argument('--beta2', type=float, default=0.999)
+parser.add_argument('--decay', type=float, default=0.01)
+
+args = parser.parse_args()
+
+url = 'https://nyu-mll.github.io/CoLA/cola_public_1.1.zip'
+
+# Download the file (if we haven't already)
+if not os.path.exists('./cola_public_1.1.zip'):
+    wget.download(url, './cola_public_1.1.zip')
+if not os.path.exists('./cola_public/'):
+    !unzip cola_public_1.1.zip
```

Comment:
This will not work in python. `!command` only workes for ipython notebooks like Jupiter notebook or google colab.

Reply:
I have made the changes.